.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug
+about: Report a bug or confusing behavior
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+Include the following information
+
+- What is the expected behavior
+- What is the actual behavior
+- Steps to reproduce
+  - Environment (CLI or web service)
```

Comment:
Can we explicitly ask for version here too?

Reply:
Done